core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -278,16 +284,31 @@ object EnumEventDescriptorV0TLV extends TLVFactory[EnumEventDescriptorV0TLV] {
     require(count.toInt == result.size,
             "Did not parse the expected number of outcomes")
 
-    EnumEventDescriptorV0TLV(result)
+    EnumEventDescriptorV0TLV(nonce, result)
   }
 }
 
-case class RangeEventDescriptorV0TLV(start: Int32, stop: Int32, step: UInt16)
+case class RangeEventDescriptorV0TLV(
+    nonce: SchnorrNonce,
+    start: Int32,
+    count: UInt32,
```

Comment:
So why is there an explicit count required? Can't this be inferred?

Reply:
If there is documentation to these classes it would be great to link to the DLC spec so it's easy for readers of your code to find things

Reply:
Can do that once it is merged

Reply:
Is there a reason you are waiting? Can't we just update links in the future? It's very hard to code review since I need to manually go find links, presumably you are following the spec as currently written?

Reply:
It is in the description of the PR

https://github.com/discreetlogcontracts/dlcspecs/pull/55/

Reply:
What section ? This is the only thing I see in the description of that PR 

>This is a proposal for oracle specification and mainly meant to serve as a basis for discussion. DO NOT MERGE

Reply:
https://github.com/discreetlogcontracts/dlcspecs/pull/55/files#diff-c15525daf7798c0dd4de183850e348ffdd9aea9812ebe5bc95a1c4189122a14dR44